packages/lodestar/src/chain/chain.ts

```diff
@@ -205,7 +205,9 @@ export class BeaconChain implements IBeaconChain {
   }
 
   public async stop(): Promise<void> {
-    this.abortController!.abort();
+    if (this.abortController) {
```

Comment:
The abortController could be instantiated in the constructor, then it would always be defined